src/components/Charts/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,35 @@
+import { LineChart as MantineLineChart } from '@mantine/charts';
+import { useViewportSize } from '@mantine/hooks';
+import { CHART_COLORS } from '@/theme';
+import { CustomLineData } from '../chart.types';
+import { chartStyles } from './styles';
+
+export default function LineChart({ data }: { data: CustomLineData }) {
+  const { width } = useViewportSize();
+  const isMobileView = width < 1024;
+
+  const { dataKey, lineValues } = data;
+  const graphData = dataKey.map((key, index) => ({
+    key,
+    ...lineValues[index].values,
+  }));
+  const seriesNames = Object.keys(lineValues[0].values);
+  const series = seriesNames.map((name, i) => ({
+    name,
+    color: CHART_COLORS[i % CHART_COLORS.length],
+    curveType: lineValues[0].curveType ?? 'linear',
+  }));
+  return (
+    <MantineLineChart
+      h={chartStyles.height(isMobileView)}
+      w={chartStyles.width}
+      data={graphData}
+      dataKey='key'
+      series={series}
+      gridProps={chartStyles.gridProps}
+      xAxisProps={chartStyles.xAxisProps}
+      yAxisProps={chartStyles.yAxisProps}
+      withTooltip={false}

```

Comment:
This should be a boolean in the chart props, you can make it optional and give it default value of false. 
We want to give control to the users of this component